woocommerce/woocommerce
# CYS: set new default patterns

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR updates the default patterns when the PTK APIs are available. These are the new default patterns:

```
header: 'woocommerce-blocks/header-essential',
intro: 'intro-centered-content-with-image-below',
footer: 'woocommerce-blocks/footer-with-3-menus',
```


![image](https://github.com/woocommerce/woocommerce/assets/4463174/4b4ad0a1-bdc6-4bbd-82b8-a4c5ca9dcf6e)





Closes #47577 #47752 #47462.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you have the last version of Gutenberg enabled
2. Make sure you have the WooCommerce beta tester plugin installed
3. Head over to Tools > WCA Test Helper > Features
4. Make sure you see the `pattern-toolkit-full-composability` on the list.
5. Enable it.
6. Head over to WooCommerce > Home > Customize your store.
7. Click on Start designing.
8. Ensure that the preview looks like the image above.
9. Click on "Design Your Homepage"
10. Click on the pattern under the header
11. Ensure that the `Shuffle` button is visible:

![image](https://github.com/woocommerce/woocommerce/assets/4463174/9847a91b-ad72-46e0-9431-63ebbef01442)






### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: set new default patterns

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
